### PR TITLE
Feedback button as in issues

### LINKED ISSUE #89: Feedback button as in issues

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
Resolve #89

### CODE REVIEW

File: src/components/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { Footer, Link } from '@taskany/bricks';
+import { FooterItem } from '@taskany/bricks/components/Footer';
+import { gray9 } from '@taskany/colors';
+
+import { tr } from './footer.i18n';
+
+export const ooter: FC = () => {
+    const menuItems = [
+        { title: tr('Terms'), url: '/terms' },
+        { title: tr('Docs'), url: '/docs' },
```
Comment (9teen90nine) [inline src/components/footer/Footer.tsx:11]: I don't think we have those pages

File: src/components/footer/footer.i18n/en.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "Terms": "Terms",
+    "Docs": "Docs",
+    "Contact Taskany": "Contact Taskany",
+    "API": "API",
+    "About": "About",
+    "Feedback": ""
```
Comment (9teen90nine) [inline src/components/footer/footer.i18n/en.json:7]: Обратная связь

File: src/components/footer/footer.i18n/ru.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "Terms": "Команды",
+    "Docs": "Документация",
+    "Contact Taskany": "Контакты",
+    "API": "API",
+    "About": "О нас",
+    "Feedback": ""
```
Comment (9teen90nine) [inline src/components/footer/footer.i18n/ru.json:7]: Обратная связь

File: src/pages/api/info.ts [hunk 4]
```diff
@@ -18,6 +18,7 @@ const handler = getApiHandler().get(async (req: NextApiRequest, res: NextApiResp
         gravatar: config.gravatar,
         sourceUsers: config.sourceUsers,
         mattermost: config.mattermost,
+        feedback: config.feedback,
```
Comment (tsumo) [inline src/pages/api/info.ts:21]: This url is sort of secret, let's not show it here

File: src/components/feedback/feedback.i18n/ru.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "Feedback title": "",
```
Comment (tsumo) [inline src/components/feedback/feedback.i18n/ru.json:2]: No translation

File: src/components/feedback/feedback.i18n/en.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+    "Feedback title": "",
+    "Feedback description. Say anything what's on your mind": "",
```
Comment (tsumo) [inline src/components/feedback/feedback.i18n/en.json:3]: Empty strings

**Review by tsumo — APPROVED** [review]

**Pull request merged**